docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "2"
+
+volumes:
+  build: ~
+  deps: ~
+
+services:
+  cabbage:
+    image: elixir:1.4
+    entrypoint: mix
+    command: do local.hex --force, deps.get, deps.compile, test
```

Comment:
`mix test` would run everything in the test suite, unit tests and integration tests. Since this is just a guide I figured it would be good to add `--only integration` so that by default it would just run all cabbage tests and not unit tests, then if they wanted to run everything they'll just remove it.